vitrina/orgs/forms.py

```diff
@@ -191,15 +192,19 @@ class Meta:
         )
 
     def __init__(self, *args, **kwargs):
+        user = kwargs.pop("user")
         super().__init__(*args, **kwargs)
-        org_instance = self.instance if self.instance and self.instance.pk else None
-        button = _("Redaguoti") if org_instance else _("Sukurti")
-        parent = self.instance.get_parent()
+
+        parent = getattr(self.instance, "get_parent", lambda: None)()
         if parent:
             self.fields["jurisdiction"].initial = parent
+
         self.helper = FormHelper()
         self.helper.attrs["novalidate"] = ""
         self.helper.form_id = "organization-form"
+        self.fields["kind"].choices = get_kind_choices(
+            user, self.instance if getattr(self.instance, "pk", None) else None
+        )
```

Comment:
Passing named variables makes it a bit more readable, since it's not clear what `self.instance` is.
```suggestion
        self.fields["kind"].choices = get_kind_choices(
            user=user, organization=self.instance if getattr(self.instance, "pk", None) else None
        )
```